README.md

```diff
@@ -1,4 +1,4 @@
-# <https://ngisweden.se>
+# [ngisweden.se](ngisweden.se)
```

Comment:
If you specifically want `http://` then I would include that in the URL..

```suggestion
# [ngisweden.se](http://ngisweden.se)
```

Reply:
Note that `<https://ngisweden.se>` is valid markdown, and shorthand for `[https://ngisweden.se](https://ngisweden.se)`

Reply:
Right, but not <ngisweden.se> 😆 ooooops!